sme_finance_application_schema/finance_application_v2

```diff
@@ -1,32 +1,37 @@
-
-{
-    "$schema":"http://json-schema.org/draft-04/schema#",
-    "title":"SME Finance Application wrapper",
-    "description":"A wrapper for an SME finance application and associated information identifying the applicant",
-    "additionalProperties": false,
-    "type": "object",
-    "required": [
-        "identifying_information", "finance_application", "unique_identifier"
-    ],
-    "properties": {
-        "identifying_information": {
-            "$ref": "https://www.fundingoptions.com/schema/sme_contact_v3#"
-        },
-        "finance_application": {
-            "oneOf": [
-                {"$ref": "https://www.fundingoptions.com/schema/sme_v4#"},
-                {"$ref": "https://www.fundingoptions.com/schema/sme_v5#"}
-            ]
-        },
-        "unique_identifier": {
-            "title":"Unique identifier",
-            "description":"A unique identifier provided by the organisation referring the applicant",
-            "type":"string"
-        },
-        "brand": {
-            "type":"string",
-            "title":"Brand",
-            "description":"The brand from which the SME has been referred (e.g. 'Natwest' for an RBS referral)"
-        }
-    }
-}
+{
+    "$schema":"http://json-schema.org/draft-04/schema#",
+    "id": "https://www.fundingoptions.com/schema/finance_application_v2",
+    "title":"SME Finance Application wrapper",
+    "description":"A wrapper for an SME finance application and associated information identifying the applicant",
+    "additionalProperties": false,
+    "type": "object",
+    "definitions": {
+        "sme_contact_v3": { "id": "sme_contact_v3"},
+        "sme_v4": { "id": "sme_v4"},
+        "sme_v5": { "id": "sme_v5"}
+    },
+    "required": [
+        "identifying_information", "finance_application", "unique_identifier"
+    ],
+    "properties": {
+        "identifying_information": {
+            "$ref": "sme_contact_v3"
+        },
+        "finance_application": {
+            "anyOf": [
```

Comment:
nit: this is technically a breaking change, however given that the original implementation was actually broken, I support this changing